editor/plugins/resource_preloader_editor_plugin.cpp

```diff
@@ -38,15 +38,17 @@
 #include "editor/editor_settings.h"
 
 void ResourcePreloaderEditor::_notification(int p_what) {
-	if (p_what == NOTIFICATION_ENTER_TREE) {
-		load->set_icon(get_theme_icon(SNAME("Folder"), SNAME("EditorIcons")));
-	}
+	switch (p_what) {
+		case NOTIFICATION_ENTER_TREE: {
+			load->set_icon(get_theme_icon(SNAME("Folder"), SNAME("EditorIcons")));
+		} break;
 
-	if (p_what == NOTIFICATION_READY) {
-		//NodePath("/root")->connect("node_removed", this,"_node_removed",Vector<Variant>(),true);
-	}
+		case NOTIFICATION_READY: {
+			//NodePath("/root")->connect("node_removed", this,"_node_removed",Vector<Variant>(),true);
+		} break;
```

Comment:
Can be removed too since it's dead code.

Reply:
Removed in latest push